0001_initial_schema.up.sql

```diff
@@ -0,0 +1,27 @@
+
+BEGIN
+
+    CREATE TABLE Clothes (
+        id SERIAL PRIMARY KEY,
```

Comment:
I looked into the SERIAL keyword. It does not guarantee a UNIQUE id, so maybe we should add the UNIQUE keyword here as well?